### ISSUE TITLE
Support for Multiple Displays

### ISSUE THREAD

**Issue by Bryan7Wilson** [kruizer23/hotframefx#13]
Created a github account specifically to ask you to add support for multiple displays. Everything else is great. Thanks for the cool app 😉

**Comment by kruizer23** [comment]
Yep, that sounds like a good idea. I could see myself use that feature too.

**Comment by LisaJackson24-2** [comment]
> Yep, that sounds like a good idea. I could see myself use that feature too.

Just great!

**Comment by Bryan7Wilson** [comment]
> Yep, that sounds like a good idea. I could see myself use that feature too.

Just great!
